packages/oui-tabs/src/item/tabs-item.controller.js

```diff
@@ -0,0 +1,64 @@
+import { addBooleanParameter, addDefaultParameter } from "@ovh-ui/common/component-utils";
+
+export default class {
+    constructor ($attrs, $element, $scope, $timeout) {
+        "ngInject";
+
+        this.$attrs = $attrs;
+        this.$element = $element;
+        this.$scope = $scope;
+        this.$timeout = $timeout;
+    }
+
+    setActive () {
+        this.tabsCtrl.activeId = this.id;
+    }
+
+    getNodeIndex () {
+        let i = 0;
+        let item = this.$element[0];
+
+        // Return DOM node index
+        while (item.previousSibling) {
```

Comment:
I find strange to see DOM manipulations and DOM acces in controller. This should be done in link function of a directive as described in angularJS best practice, no?

Reply:
It's more DOM navigation than DOM manipulation, but we could put that part in $postLink :)